sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -456,7 +456,7 @@ export function extractReceiverArguments<OptionsT extends { receiveMode?: Receiv
     options = optionsOrSubscriptionName2;
   }
   let receiveMode: ReceiveMode;
-  if (options?.receiveMode == undefined || options.receiveMode === "peekLock") {
+  if (options?.receiveMode === undefined || options.receiveMode === "peekLock") {
```

Comment:
No null check?

Reply:
We don't expect the user to pass null, but we can expect the user to not set `options.receiveMode` to anything at all. So, the check on undefined is justified, but not for null. Another way to do this would be to use the `Object.hasOwnProperty`  method

Reply:
Updated using the approach @chradek has for Event Hubs. See https://github.com/Azure/azure-sdk-for-js/pull/13935/commits/29e0618fd3e30f31f760eacf909aa9c2cdfce6b2